components/Downloads/Release/BitnessDropdown.tsx

```diff
@@ -52,6 +52,10 @@ const BitnessDropdown: FC = () => {
       disabledItems.push('s390x');
     }
 
+    if (os === 'AIX' && semVer.satisfies(release.version, '< 4.1.0')) {
```

Comment:
```suggestion
    if (os === 'AIX' && semVer.satisfies(release.version, '< 6.17.1')) {
```

I think we don't support it for the versions below, when I go to the download links it redirects to a 404 page;

  - v4.9.1 (https://nodejs.org/dist/v4.9.1/node-v4.9.1-aix-ppc64.tar.gz)
  - v5.12.0 (https://nodejs.org/dist/v5.12.0/node-v5.12.0-aix-ppc64.tar.gz)

Reply:
According to [index.json](https://nodejs.org/dist/index.json) / [index.tab](https://nodejs.org/dist/index.tab), the first AIX binaries are Node.js 6.7.0 (https://nodejs.org/dist/v6.7.0/node-v6.7.0-aix-ppc64.tar.gz).